keps/sig-storage/1710-selinux-relabeling/README.md

```diff
@@ -446,19 +441,26 @@ _This section must be completed when targeting beta graduation to a release._
   Try to be as paranoid as possible - e.g. what if some components will restart
   in the middle of rollout?
   
-  Running workloads are not affected during rollout, because they don't use the new API fields. 
+  This KEP affects only kubelet behavior and only mounts on the node where kubelet runs.
+  Different nodes in a cluster can have the feature enabled/disabled without any issues.
 
 * **What specific metrics should inform a rollback?**
 
+  TBD: We propose a metric above, file its name here.
+  
 * **Were upgrade and rollback tested? Was upgrade->downgrade->upgrade path tested?**
   Describe manual testing that was done and the outcomes.
   Longer term, we may want to require automated upgrade/rollback tests, but we
   are missing a bunch of machinery and tooling and do that now.
 
+  TBD: this must be tested probably manually.
+
 * **Is the rollout accompanied by any deprecations and/or removals of features,
   APIs, fields of API types, flags, etc.?**
   Even if applying deprecation policies, they may still surprise some users.
 
+  **YES!** See [Behavioral changes](#behavioral-changes) and [Implementation phases](implementation-phases) above.
```

Comment:
The new failure mode looks slightly better to my eyes, but it still looks difficult to debug as a pod author.

Reply:
I improved the error messages where kubelet can know what's wrong.